.gitignore

```diff
@@ -5,3 +5,4 @@ coverage
 *.sw?
 .DS_Store
 runtime.json
+.idea
```

Comment:
I think it comes handy if one is using one of these IDEs. Would you rather see it in a separate PR or not at all? :)